ws-nextjs-app/pages/[service]/send/[id]/FormField/FieldLabel.tsx

```diff
@@ -9,8 +9,22 @@ export default ({
   id,
   children,
   className,
-}: PropsWithChildren<{ id: InputProps['id']; className?: string }>) => (
-  <Text as="label" className={className} htmlFor={id} css={styles.fieldLabel}>
+  withPTag,
+}: PropsWithChildren<{
+  id: InputProps['id'];
+  className?: string;
+  withPTag?: boolean;
+}>) => (
+  <Text
+    as="label"
+    className={className}
+    htmlFor={id}
+    css={({ spacings }: Theme) =>
+      withPTag
+        ? styles.fieldLabel
+        : [styles.fieldLabel, `marginBottom: ${spacings.FULL}rem`]
```

Comment:
I don't think we need this logic. If we retain the `marginBottom` that was on the `fieldLabel` CSS and then change the textarea `Paragraph` component to `css={{ marginBottom: `${pixelsToRem(6)}rem` }}`, I think that should achieve the same thing?